devpack/src/main/java/io/neow3j/devpack/Storage.java

```diff
@@ -80,16 +80,6 @@
     @Syscall(SYSTEM_STORAGE_PUT)
     public static native void put(StorageContext context, byte[] key, int value);
 
-    /**
-     * Stores the given key-value pair.
-     *
-     * @param context The storage context to store the value in.
-     * @param key     The key.
-     * @param value   The value to store.
-     */
-    @Syscall(SYSTEM_STORAGE_PUT)
-    public static native void put(StorageContext context, byte[] key, long value);
```

Comment:
@csmuller the only thing about this pull request that I didn't get, is: why did you decide to remove these methods in the `Storage.java` class?

Reply:
They are duplicates of the same methods with the only difference of using `int` instead of `long` as the parameters. Since the goal of this issue/PR was to switch everything to `int`, I decided to remove the `long`-methods.